pdc/apps/common/renderers.py

```diff
@@ -140,6 +141,11 @@ def format_docstring(self, view, method, docstring):
         macros = settings.BROWSABLE_DOCUMENT_MACROS
         if view:
             macros['FILTERS'] = get_filters(view)
+            if 'list' == method:
+                ordering_field = get_ordering_field(view, method)
+                if ordering_field:
+                    ordering_string = ORDERING_STRING + "\n %s ." % ordering_field
```

Comment:
It is better to add markdown format. E.g surround each field with `` ` `` could make it read, etc. And it looks same with or without above `\n`.